package.json

```diff
@@ -60,6 +60,7 @@
   "scripts": {
     "lint": "balena-lint -e js --typescript lib",
     "lint-fix": "balena-lint -e js --typescript --fix lib",
+    "lint-python": "flake8 src/preload.py",
```

Comment:
This `npm` script was added as a "hint" for someone to execute it manually if they know and remember about it. Alexis mentioned it to me in this older PR: https://github.com/balena-io-modules/balena-preload/pull/220#pullrequestreview-446302471

Automated execution is a "To Do". The `flake8` command is a python utility, not `npm` package, so cannot (?) be installed with something like "npm install", and adding some sort of "pip install" would mean having to deal with setting up a python environment or Dockerfile.